docs/src/pages/style/links/links.md

```diff
@@ -0,0 +1,38 @@
+---
+components: Link
+---
+
+# Links
+
+<p class="description">The Link component creates a hyperlink to other web pages, files, locations within the same page, email addresses, or any other URL.</p>
+
+## Simple links
+
+The Link component is built on top of the [Typography](/api/typography/) component.
+You can leverage its properties.
+
+{{"demo": "pages/style/links/Links.js"}}
+
+For the best user experience links should stand out from the text on the page.
+
+{{"demo": "pages/style/links/LinkUserExperience.js"}}
```

Comment:
What's the purpose of this demo? The (non-inherit) links in the previous example already demonstrate this feature. It seems a bit redundant.

Reply:
It was more for accessibility, I've moved the sentence into the Accessibility section and removed the demo